Last-Update: 2013-09-11
Forwarded: yes
Applied-Upstream: https://github.com/jpirko/libteam/commit/b68f096dec08e8e02e06d1cbf6b902c88051da38
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Description: minor man page(s) correction(s) and lintianisation
 manpage-has-errors-from-man usr/share/man/man1/bond2team.1.gz 75: warning: macro `ni' not defined
 hyphen-used-as-minus-sign

--- a/man/bond2team.1
+++ b/man/bond2team.1
@@ -71,14 +71,14 @@
 .fi
 .PP
 To convert given bonding parameters with ports without any ifcfg:
 .PP
-.ni
-# bond2team --bonding_opts '\%mode=1 \%miimon=500 \%primary=eth1 \%primary_reselect=0' \%--port \%eth1 \%--port \%eth2 \%--port \%eth3 \%--port \%eth4
+.nf
+# bond2team \-\-bonding_opts '\%mode=1 \%miimon=500 \%primary=eth1 \%primary_reselect=0' \%--port \%eth1 \%--port \%eth2 \%--port \%eth3 \%--port \%eth4
 .fi
 .PP
 For more examples, see the option
-.B --examples.
+.B \-\-examples.
 .SH CAVEATS
 .PP
 The tool will not convert any other configuration which might be tied to the current setup. For instance, firewall rules, aliases interfaces, bridges, and etc.
 .SH AUTHOR
